libr/include/r_io.h

```diff
@@ -145,6 +145,10 @@ typedef struct r_io_t {
 	// TODO: rename to dbgwrap
 	struct ptrace_wrap_instance_t *ptrace_wrap;
 #endif
+void *origread;
```

Comment:
none of those vars belong to the IO api

Reply:
I will move these vars to the same macho specific struct that I will move `heapplugin` to.